suite-native/module-accounts-import/src/components/SelectableAssetItem.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { CryptoIcon, CryptoIconName } from '@trezor/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Box, Button, Text } from '@suite-native/atoms';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { useFormatters } from '@suite-common/formatters';
+
+export type SelectableAssetItemProps = {
+    cryptoCurrencySymbol: NetworkSymbol;
+    cryptoCurrencyName: string;
+    iconName: CryptoIconName;
+    onPress?: () => void;
+    onPressActionButton?: () => void;
+};
+
+const selectableAssetContentStyle = prepareNativeStyle(() => ({
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    flex: 1,
+    marginLeft: 12,
+}));
+
+const cryptoIconStyle = prepareNativeStyle(utils => ({
+    height: 48,
+    width: 48,
+    borderRadius: utils.borders.radii.round,
+    backgroundColor: utils.colors.gray200,
+}));
+
+export const SelectableAssetItem = ({
+    cryptoCurrencyName,
+    cryptoCurrencySymbol,
+    iconName,
+    onPress,
+    onPressActionButton,
+}: SelectableAssetItemProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const { CurrencySymbolFormatter } = useFormatters();
+
+    return (
+        <TouchableOpacity disabled={!onPress} onPress={onPress}>
+            <Box flexDirection="row" alignItems="center">
+                <Box justifyContent="center" alignItems="center">
+                    <Box
+                        justifyContent="center"
+                        alignItems="center"
```

Comment:
move to `cryptoIconStyle` since style object is present to have all styles together

Reply:
e21e5713ac754d22544aedeb5ae901093cf708a7